pages/people/[id]/index.tsx

```diff
@@ -11,24 +11,29 @@ import Stack from 'components/Stack/Stack';
 import { useAuth } from 'components/UserContext/UserContext';
 import { User } from 'types';
 
+const canViewWarningNotes = (user: User) => {
```

Comment:
Would this mean only users with `dev permissions` can view warning notes?

Reply:
That's right – I actually just refactored a previous check that was already doing this inline here: https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/354/files#diff-b5ded0b0b90dbeccfcd0da5f2457c48db56fb45bcccbe174155aa7081e98ce43L29

Reply:
I see 👍🏾